packages/adminrouter/extra/src/includes/server/open/master.conf

```diff
@@ -1,6 +1,11 @@
-# Group: Pkgpanda
+#IAtrea Group: Pkgpanda
```

Comment:
What does this change do?

Reply:
Seems like a typo, should be [iatreia](http://aprendeenlinea.udea.edu.co/revistas/index.php/iatreia)

Reply:
Thanks for spotting this @jongiddy . Lol @jgehrcke